source/image/ImageSet.cpp

```diff
@@ -187,13 +187,14 @@ void ImageSet::Load() noexcept(false)
 	// to be in separate locations on the disk. Create masks if needed.
 	for(size_t i = 0; i < frames; ++i)
 	{
+		const string fileName = "\"" + name + "\" frame #" + to_string(i);
```

Comment:
Should we just drop the "frame #" part if there is only one frame?
```suggestion
		const string fileName = "\"" + name + (frames > 1 ? "\" frame #" + to_string(i) : "\"");
```

Reply:
it's good as-is